SwiftUiMovieDb/Views/MoviePosterCard.swift

```diff
@@ -6,35 +6,33 @@
 //
 
 import SwiftUI
+import Kingfisher
 
 struct MoviePosterCard: View {
     
     let movie: Movie
-    
-    @ObservedObject var imageLoader = ImageLoader()
-    
+    let cache = ImageCache.default
     var body: some View {
         ZStack {
-            if self.imageLoader.image != nil {
-                Image(uiImage: self.imageLoader.image!)
-                    .resizable()
-                    .aspectRatio(contentMode: .fill)
-                    .cornerRadius(8)
-                    .shadow(radius: 4)
-            } else {
+            if cache.isCached(forKey: "my_cache_key") {
```

Comment:
I don't this is needed anymore, instead you can use **placeholder** modifier on KFImage itself.